### PR TITLE
Bug 1834293: Unnecessary optional chaining after ternary operator check

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
/assign @jcaianirh
 /cc @benjaminapetersen

### CODE REVIEW

File: frontend/public/components/persistent-volume-claim.jsx [hunk 1]
```diff
@@ -127,8 +127,8 @@ const PVCTableRow = ({ obj, index, key, style }) => {
         {obj?.spec?.storageClassName ? (
           <ResourceLink
             kind="StorageClass"
-            name={obj?.spec?.storageClassName}
-            title={obj?.spec?.storageClassName}
+            name={obj.spec.storageClassName}
+            title={obj.spec.storageClassName}
           />
         ) : (
           <div className="text-muted">-</div>
```
Comment (spadgett) [inline frontend/public/components/persistent-volume-claim.jsx:134]: Can you remove the `text-muted` here while in the file? It's inconsistent with other `-` cells in the same table that don't have the `text-muted` style.

Comment (cyril-ui-developer) [reply frontend/public/components/persistent-volume-claim.jsx]: > Can you remove the `text-muted` here while in the file? It's inconsistent with other `-` cells in the same table that don't have the `text-muted` style.

Done. Thanks.

File: frontend/public/components/persistent-volume-claim.jsx [hunk 2]
```diff
@@ -127,11 +127,11 @@ const PVCTableRow = ({ obj, index, key, style }) => {
...
           <ResourceLink
             kind="StorageClass"
-            name={obj?.spec?.storageClassName}
-            title={obj?.spec?.storageClassName}
+            name={obj.spec.storageClassName}
+            title={obj.spec.storageClassName}
           />
         ) : (
-          <div className="text-muted">-</div>
+          <div>-</div>
```
Comment (spadgett) [inline frontend/public/components/persistent-volume-claim.jsx:134]: This div should be unnecessary.

```suggestion
          <>-</>
```

Comment (cyril-ui-developer) [reply frontend/public/components/persistent-volume-claim.jsx]: > This div should be unnecessary.

Got it, thanks

**Pull request merged**